packages/scene-composer/src/augmentations/components/three-fiber/anchor/AnchorWidget.tsx

```diff
@@ -77,8 +85,11 @@ export function AsyncLoadedAnchorWidget({
 
   const baseScale = useMemo(() => {
     // NOTE: For Fixed Size value was [0.05, 0.05, 1]
-    return new THREE.Vector3(0.5, 0.5, 1).multiply(new THREE.Vector3(...node.transform.scale));
-  }, [node.transform.scale]);
+    const defaultScale = autoRescale ? [0.05, 0.05, 1] : [0.5, 0.5, 1];
```

Comment:
How did we come up with these numbers>

Reply:
from what we currently use and what's in the note in the comment above

Reply:
What if we change these values to random numbers? Do we need a strict check on this?